tools/ci/docs.sh

```diff
@@ -17,29 +17,22 @@
 # limitations under the License.
 ################################################################################
 
-# fail immediately
-set -o errexit
-set -o nounset
+HUGO_REPO=https://github.com/gohugoio/hugo/releases/download/v0.80.0/hugo_extended_0.80.0_Linux-64bit.tar.gz
+HUGO_ARTIFACT=hugo_extended_0.80.0_Linux-64bit.tar.gz
 
-BASE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
-PROJECT_ROOT="${BASE_DIR}/../../"
-
-# Sanity check to ensure that resolved paths are valid
-if [ ! -f ${PROJECT_ROOT}/docs/build_docs.sh ]; then
-    echo "Project root path ${PROJECT_ROOT} is not valid; script may be in the wrong directory."
-    exit 1
+if ! curl --fail -OL $HUGO_REPO ; then 
+	echo "Failed to download Hugo binary"
+	exit 1
 fi
 
-CACHE_DIR=$HOME/gem_cache ${PROJECT_ROOT}/docs/build_docs.sh -p &
+tar -zxvf $HUGO_ARTIFACT
```

Comment:
I think it would be safer to extract the hugo artifact in a separate directory, just in case it somehow pollutes the docs dir. I don't think that's the case right now, but would be good to do.

Reply:
I've updated it to only extract the specific binary file we need.